doc/api/child_process.markdown

```diff
@@ -729,10 +729,30 @@ an error but may have unforeseen consequences. Specifically, if the process
 identifier (PID) has been reassigned to another process, the signal will be
 delivered to that process instead which can have unexpected results.
 
+See `kill(2)` for general reference.
+
 Note that while the function is called `kill`, the signal delivered to the
-child process may not actually terminate the process.
+child process may not actually terminate the process. E.g. on `linux` child
```

Comment:
Any reason why `linux` is inside a code segment here rather than being capitalized? (Linux)